src/backend.rs

```diff
@@ -55,13 +55,13 @@ impl<'a> CairoBackend<'a> {
         self.context.set_font_size(font.size());
     }
 
-    pub fn new(context: &'a CairoContext, (w, h): (u32, u32)) -> Self {
-        Self {
+    pub fn new(context: &'a CairoContext, (w, h): (u32, u32)) -> Result<Self, CairoError> {
```

Comment:
error[E0412]: cannot find type `CairoError` in this scope

Should be cairo::Error? I will commit a fix.